nucleus/model.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, Optional, Dict, Union
+from .dataset import Dataset
+from .prediction import BoxPrediction, PolygonPrediction
+from .model_run import ModelRun
+
+
+class Model:
+    def __init__(
+        self,
+        model_id: str,
+        name: str,
+        reference_id: str,
+        metadata: Optional[Dict],
+        client,
+    ):
+        self.id = model_id
+        self.name = name
+        self.reference_id = reference_id
+        self.metadata = metadata
+        self._client = client
+
+    def create_run(
+        self,
+        name: str,
+        metadata: dict,
+        dataset: Dataset,
+        predictions: List[Union[BoxPrediction, PolygonPrediction]],
+    ) -> ModelRun:
+        payload = {
+            "name": name,
+            "reference_id": self.reference_id,
+            "metadata": metadata,
+        }
+        model_run: ModelRun = self._client.create_model_run(
+            dataset.id, payload
+        )
+
+        model_run.predict(predictions)
+
+        return model_run
```

Comment:
I'm worried that adding a model_run as a property will be a bit misleading, because we don't automatically pull in all of the model_runs associated with this model (other runs could have been created previously).  I think the problem you're pointing out here is super valid though: how does a user fetch all model_runs associated with a model?  In the future we should add API endpoints for:
- get_model
- delete model
- get_model_runs for model